docs/source/index.rst

```diff
@@ -82,6 +82,7 @@ What are the full benefits when I reach Level 5?
 - Streamline the release process by pushing a Git tag to trigger a sequence of actions: publishing to PyPI and GitHub, updating hosted documentation, and updating the ``CHANGELOG.rst`` file.
 - Host documentation with a public URL using a ``Sphinx`` template. Include live rendering, API documentation, and previews for each pull request.
 - Provide a rich ``README.rst`` template that includes badges, installation instructions, support contacts, and contribution guidelines for your GitHub repository.
+- Set custom default metadata such as authors, project description, license holders, and keywords when creating a new package. Automatically store and reuse the entered metadata to update your package with the latest version of the ``scikit-package`` standards.
```

Comment:
do we want to link to where in the docs this is described?   I guess this isn't done for the other bullets.    The read-the-docs template gives a little search box if I remember well, so just giving sometihng users can copy into there to find it would be good.   What would that be?  Something like "user config" or sthg?